sklearnex/tests/test_common.py

```diff
@@ -49,6 +49,54 @@
     "IncrementalEmpiricalCovariance-score-call_validate_data",  #  must call clone of itself
     "SVC(probability=True)-fit-call_validate_data",  #  SVC fit can use sklearn estimator
     "NuSVC(probability=True)-fit-call_validate_data",  #  NuSVC fit can use sklearn estimator
+    "LogisticRegression-score-n_jobs_check",  # uses daal4py for cpu in sklearnex
```

Comment:
We only have one list for all design violations vs all design rules?
Should be renamed for the specific rule I think and better to have for the specific rule specific maybe list of design violation.

Reply:
The design rule is in the string, I'm not sure I understand

Reply:
I mean we have only one skip list for the all design rules?

Reply:
I mean test should be parameterized like this
`def test_estimator(estimator, method, design_pattern, design_violation_list, estimator_trace):
`
Probably some design patterns should have some part of common design violation list, that should be defined.
But if we expect that all rules will not have the specific violations except some common list then make sense to leave as is

Reply:
But probably the comment for the further update :)

Reply:
Thank you @icfaust !
We have discussed it somehow, assuming we make the comment resolved.
We have one skip list for each test cases, where the design_rule is specified. Im good with this one.

Just for the further optimization we can add the string comment into pytest,xfail. to see this in the logs not only in code base. But this refactoring not the scope of the current PR.

Reply:
Done, it will now give an informative string on xfail.